qiime2/core/cache.py

```diff
@@ -1194,7 +1194,11 @@ def _alias(self, uuid):
 
             # Named pool links are not aliased
             if self.named_pool is not None:
-                self.named_pool._make_symlink(uuid, uuid)
+                if not os.path.exists(self.named_pool.path):
+                    warnings.warn("The named pool path"
```

Comment:
I don't think we actually want to set named pool to None after this warning because it has potential knock on effects later down the line when other bits of code are expecting there to not be a none here